examples/phi2/phi2.py

```diff
@@ -112,10 +118,10 @@ def get_output_model_path(footprints):
 
 def main(raw_args=None):
     args = get_args(raw_args)
-    if not args.model_type and not args.finetune_method:
-        raise ValueError("Please specify either model_type or finetune_method")
+    if not args.model_type and not args.finetune_method and not args.slicegpt:
```

Comment:
I think we should update the condition here to make sure only one of these is provided at one time since they are supposed to be mutually exclusive. 

This way, we can use the mutually exclusive fact to simplify the conditions in the following code. Like I described in my other comments.

Reply:
Acutually never mind. model_type and finetune_method are compatable with one another.